pkg/cmd/volume/rm.go

```diff
@@ -44,23 +46,28 @@ func Remove(ctx context.Context, client *containerd.Client, volumes []string, op
 
 	var volumenames []string // nolint: prealloc
 	for _, name := range volumes {
-		volume, err := volStore.Get(name, false)
-		if err != nil {
-			return err
-		}
-		if _, ok := usedVolumes[volume.Name]; ok {
+		if _, ok := usedVolumes[name]; ok {
 			return fmt.Errorf("volume %q is in use", name)
 		}
 		volumenames = append(volumenames, name)
 	}
-	removedNames, err := volStore.Remove(volumenames)
+	// if err is set, this is a hard filesystem error
+	removedNames, errs, err := volStore.Remove(volumenames)
 	if err != nil {
 		return err
 	}
+	// Otherwise, output on stdout whatever was successful
```

Comment:
This and following is to match docker behavior on multi remove:
- first output the list of successfully removed volumes
- then output the errors
- exit 1 if there was any error